pythran/config.py

```diff
@@ -344,6 +358,26 @@ def run():
 
     args = parser.parse_args(sys.argv[1:])
 
+    # This should not rely on distutils/setuptools, or anything else not in the
+    # stdlib. Please don't add imports higher up.
+    if args.cflags_pythran_only:
+        if args.cflags:
+            print("Error: --cflags and --cflags-pythran-only are mutually "
+                  "exclusive, please use only one.")
+
+        compile_flags = [
+                "-D__PYTHRAN__=3",
+                "-DENABLE_PYTHON_MODULE",
+                "-DPYTHRAN_BLAS_NONE",
```

Comment:
I understand that this blas flag is the one you need for scipy, but it should really depend on current config.

You mentioned

> I considered adding --config compiler.blas=none|openblas support, but that then also has to interact with --cflags etc., so it's not clear that that works well for inclusion in 0.17.1 - and this can be added later.

In the gommit message but I think that's fine, it should just be also forwaded to extension = `pythran.config.make_extension(python=args.python)` below

Reply:
Okay thanks, I'll see if I can figure out how to introduce this now then.